### ISSUE TITLE
Sound Filter on/off Request

### ISSUE THREAD

**Issue by Nashismo** [dborth/snes9xgx#751]
Hi Dborth:

First let me thank you for your wonderfull work! The last version of the emu (4.3.9) is running flawlessly for me, no sound stutters, no sound distortion, excelent speed!

But, I have been wondering what could be the reason why the sound still feels a bit better on Snes9x (retroarch Wii), at first I thought that it was because of filters, but now that I have been playing with audio filters (in Genesis Plus GX) I realize that maybe the issue I am seeing, is that Snes9xGx actually "does" use a filter (and Snes9x Retroarch core does not).

You see I would love to have a more "raw" sound for the emu. Maybe you can give us an option to have audio filtering on or off, so we could use it to our liking.

Or maybe there is a way to disable/enable audio filtering using the xml file?

Thank you for reading dborth, thank you very much for your time.

**Comment by Nashismo** [comment]
Now I have made various tests and Snes9x (retroarch) sounds more punchy and better for me, because I was using linear sound filtering. Maybe you are using Gaussian filtering maybe? To me linear filtering is the best because it keeps that raw sound while still serving it's purpose as a sound filter (gaussian and others are too much).

If you read this, please do tell which sound filtering Snes9xGX is using, and if so, how to change it please. Thank you Dborth!

**Comment by Nashismo** [comment]
I did some more testing and it actually the emu sounds quite good anyways, better than the other versions (I was also using older versions to compare back and forward).
The thing is, my request is still there just in case you want to add something in the future (but it is not something that important).

Thanks again.

**Comment by dborth** [comment]
As it is now, it's using a gaussian filter because that's what the original SNES used. So the sound is true to the original hardware. It's possible to add a setting to disable this and just output the raw sound. That wouldn't actually be true to the hardware, but it's possible.

https://www.reddit.com/r/retrogaming/comments/7hki82/eli5_why_does_snes_music_sound_muffled_compared/
